trunk/SUAVE/Methods/Aerodynamics/Airfoil_Panel_Method/thwaites_method.py

```diff
@@ -0,0 +1,292 @@
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+# thwaites_method.py 
+
+# Created:  Mar 2021, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+from SUAVE.Core import Data 
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.integrate import odeint 
+
+# ----------------------------------------------------------------------
+# thwaites_method
+# ----------------------------------------------------------------------  
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+def thwaites_method(nalpha,nRe,L,RE_L,X_I,VE_I, DVE_I,batch_analysis,THETA_0,n = 200,tol  = 1E0):
```

Comment:
Yes, I tried different tolerances. This states that if the successive values are more than 100% i.e. if it blows up where H is nonsensical, replace the value. As I mentioned, there are multiple ways of computing H. This is not the best but is widely used because of ease of implementation.